spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -302,6 +315,11 @@ def main(argv=None):
             spaces.append(None)
         im_out = [ displacement_to_abs_fsl(Image(fname_in[0]), spaces[0], spaces[1]) ]
 
+    elif arguments.set_sform_to_qform is not None:
+        # This argument can be used stand alone or with other operations so
+        # it should go last before the else
+        im_out = [Image(fname_in[0])]
```

Comment:
Then, we could call the API method from `sct_image`.

```suggestion
    elif arguments.set_sform_to_qform is not None:
        # This argument can be used stand alone or with other operations so
        # it should go last before the else
        im_out = [Image(fname_in[0]).set_sform_to_qform()]
```